src/openeo_gfmap/manager/job_manager.py

```diff
@@ -53,11 +90,14 @@ def __init__(
         post_job_action: Optional[Callable] = None,
         poll_sleep: int = 5,
         n_threads: int = 1,
-        post_job_params: dict = {},
         resume_postproc: bool = True,  # If we need to check for post-job actions that crashed
         restart_failed: bool = False,  # If we need to restart failed jobs
```

Comment:
What is the difference here between restarting and resuming? Do you have everything up to the post processing cached?

Reply:
I removed the parameter `post_job_params` here because it is not being used. Instead, in order to parametrize my `post_job_action` I make the use of the `functools.partial` feature.

My extraction scripts always set-up the same parameters for the post-job-action between starting fresh/resuming, but I agree that it gives responsibility to our users

Reply:
As an example: https://github.com/WorldCereal/worldcereal-classification/blob/main/scripts/extractions/patch_extractions/extract.py#L173